helix-term/Cargo.toml

```diff
@@ -50,3 +50,6 @@ toml = "0.5"
 
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
+
+unicode-segmentation = "1.7"
```

Comment:
I should re-export all three unicode crates from `core` I think.

`core::unicode::width/segmentation/category`

WDYT?